dist/windows/installer.nsi

```diff
@@ -32,27 +32,7 @@ Section $(inst_qbt_req) ;"qBittorrent (required)"
   File "qbittorrent.exe"
   File "qbittorrent.pdb"
   File "qt.conf"
-  File /oname=translations\qtbase_ar.qm "translations\qtbase_ar.qm"
-  File /oname=translations\qtbase_bg.qm "translations\qtbase_bg.qm"
-  File /oname=translations\qtbase_ca.qm "translations\qtbase_ca.qm"
-  File /oname=translations\qtbase_cs.qm "translations\qtbase_cs.qm"
-  File /oname=translations\qtbase_da.qm "translations\qtbase_da.qm"
-  File /oname=translations\qtbase_de.qm "translations\qtbase_de.qm"
-  File /oname=translations\qtbase_es.qm "translations\qtbase_es.qm"
-  File /oname=translations\qtbase_fi.qm "translations\qtbase_fi.qm"
-  File /oname=translations\qtbase_fr.qm "translations\qtbase_fr.qm"
-  File /oname=translations\qtbase_gd.qm "translations\qtbase_gd.qm"
-  File /oname=translations\qtbase_he.qm "translations\qtbase_he.qm"
-  File /oname=translations\qtbase_hu.qm "translations\qtbase_hu.qm"
-  File /oname=translations\qtbase_it.qm "translations\qtbase_it.qm"
-  File /oname=translations\qtbase_ja.qm "translations\qtbase_ja.qm"
-  File /oname=translations\qtbase_ko.qm "translations\qtbase_ko.qm"
-  File /oname=translations\qtbase_lv.qm "translations\qtbase_lv.qm"
-  File /oname=translations\qtbase_pl.qm "translations\qtbase_pl.qm"
-  File /oname=translations\qtbase_ru.qm "translations\qtbase_ru.qm"
-  File /oname=translations\qtbase_sk.qm "translations\qtbase_sk.qm"
-  File /oname=translations\qtbase_uk.qm "translations\qtbase_uk.qm"
-  File /oname=translations\qtbase_zh_TW.qm "translations\qtbase_zh_TW.qm"
+  File /r "qtbase_*.qm"  ; omit translations folder path to preserve folder structure
```

Comment:
This will recursively find all matched files for `qtbase_*.qm` pattern. For example the source is "translations\qtbase_ar.qm", then the target will be the same "translations\qtbase_ar.qm", i.e. the sub folder will be preserved.

Reply:
Why not `File /r "translations\qtbase_*.qm"`? (to be more specific)

Reply:
Then the `qtbase_*.qm` files will be placed at the same level as `qbittorrent.exe` which is incorrect.

Reply:
OK. I understood.
Btw, from the reading the docs, this should work:
```
SetOutPath $INSTDIR\translations
File /r "translations\qtbase_*.qm"
SetOutPath $INSTDIR

// rest of the files
```

But we probably don't want this.